staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere.go

```diff
@@ -1221,9 +1221,15 @@ func (vs *VSphere) DisksAreAttached(nodeVolumes map[k8stypes.NodeName][]string)
 			}
 
 		}
+		klog.V(4).Infof("DisksAreAttach successfully executed. result: %+v", attached)
```

Comment:
nit: DisksAreAttached

Reply:
fixed. (in my defence I just moved existing log. :-) )